pkg/build/registry/buildlog/rest.go

```diff
@@ -120,6 +121,8 @@ func (r *REST) waitForBuild(ctx kapi.Context, build *api.Build) error {
 	done := make(chan struct{})
 	errchan := make(chan error)
 	go func(ch <-chan watch.Event) {
+		defer util.HandleCrash()
```

Comment:
@smarterclayton suggested this into our `waitForDeployment` function so I added it to `waitForBuild` too since it has the same chances to panic